samples/maps/demo/flowmap-around-the-world/demo.html

```diff
@@ -0,0 +1,7 @@
+<script src="https://code.highcharts.com/maps/highmaps.js"></script>
+<script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/maps/modules/offline-exporting.js"></script>
+<script src="https://code.highcharts.com/maps/modules/sankey.js"></script>
+<script src="https://code.highcharts.com/maps/modules/flowmap.js"></script>
+
```

Comment:
Also, what about the accessibility module? Do we use it here, or it's not necessary?